README.md

```diff
@@ -0,0 +1,58 @@
+# OpenID for Verifiable Presentations (OID4VP)
+
+[![Crates.io](https://img.shields.io/crates/v/oid4vp)](https://crates.io/crates/oid4vp)
+[![Docs.rs](https://docs.rs/oid4vp/badge.svg)](https://docs.rs/oid4vp)
+[![License](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
+
+### Rust implementation of the OpenID for Verifiable Presentations (OID4VP) specification.
+
+
+## Install
+
+Add the following to your `Cargo.toml`:
+
+```toml
+[dependencies]
+oid4vp = "0.1"
+```
+
+or
+
+```shell
+cargo add oid4vp
+```
+
```

Comment:
We will have to rename before we publish -- someone else has already squatted on the name `oid4vp`.